course_1_1/Program.cs

```diff
@@ -10,594 +10,121 @@ class Program
-                Console.WriteLine("Add number B");
-                int numberB = Convert.ToInt32(Console.ReadLine());
-                if (numberA > numberB)
-                {
-                    Console.WriteLine("Initial number is greater than the final. Please try again");
-                    check = true;
-                    continue;
-                }
-                int sum = 0;
-                int counter = 0;
-                int average;
-                for (int i = numberA; i <= numberB; i++)
-                {
-                    sum += i;
-                    counter++;
-                }
-                average = sum / counter;
-                Console.WriteLine($"Average = {average} \nInput Y to repeat");
-                string repeat= Console.ReadLine();
-                if (repeat == "Y")
-                {
-                    check = true;
-                }
-                else
-                {
-                    check = false;
-                }
-            } while (check);
-           
-        }
-        public static void RandomNumber()
-        {
-            bool check = false;
-            int number = new Random().Next(1,100);
-            int ourNumber = 0;
-            do
-            {
-                Console.WriteLine("Enter Number:");
-                ourNumber = Convert.ToInt32(Console.ReadLine());
-                if (ourNumber > number)
-                {
-                    Console.WriteLine("Less");
-                    check = true;
-                }
-                else if (ourNumber < number)
-                {
-                    Console.WriteLine("More");
-                    check = true;
-                }
-                else
-                {
-                    Console.WriteLine($"Congratulations! Random number is {number}");
-                    check = false;
-                } 
-            } while (check);         
-        }
-        public static void Exponentiation()
-        {
-            Console.WriteLine("Add number");
-            int number = Convert.ToInt32(Console.ReadLine());
-            Console.WriteLine("Add exponent");
-            int exponent = Convert.ToInt32(Console.ReadLine());
-            int result=1;
-            for (int i = 1; i <= exponent; i++)
-            {
-                result *= number;
-            }   
-            Console.WriteLine(result);
-        }
-        public static void ProductRoute()
-        {
-            Console.WriteLine("Add number of clients");
-            int clients = Convert.ToInt32(Console.ReadLine());
-            int n=1;
-            int result=1;
-            do
-            {
-                result *= n;
-                n++;
-            } while (n<=clients);
-            Console.WriteLine($"Number of possible routes: {result}");
+            //ArrayStore();
+
+            ArrayEncoder();
         }
-        public static void DrowFigure()
+        public static void ArrayUnpacking()
```

Comment:
The task is not completed.
According to the description the logic should unpack array of any dimension.